lib/src/request/CommonMediaResponse.ts

```diff
@@ -0,0 +1,55 @@
+import { CommonMediaRequest } from './CommonMediaRequest.js';
+import { ResourceTiming } from './ResourceTiming.js';
+
+/**
+ * Common response API.
+ * 
+ * @group Request
+ */
+
+export interface CommonMediaResponse {
+  /**
+   * The origin request.
+   */
+  request: CommonMediaRequest
+
+  /**
+   * The final URL obtained after any redirects.
+   */
+  url?: string
+
+  /**
+   * Indicates whether or not the request was redirected.
+   */
+  redirected?: boolean
+
+  /**
+   * The HTTP status code of the response.
+   */
+  status?: number
+
+  /**
+   * The status message corresponding to the HTTP status code.
+   */
+  statusText?: string
+
+  /**
+   * The type of the response.
+   */
+  type?: string
+
+  /**
+   * The response headers.
+   */
+  headers?: Record<string, string>
+
+  /**
+   * The response data.
+   */
+  data?: any
+
+  /**
+   * The network timing of the request/response.
+   */
+  resourceTiming: ResourceTiming
```

Comment:
Do we need to define our own interface, or can we use `PerformanceResourceTiming` directly?

Reply:
The point is that you can't create a `PerformanceResourceTiming` object instance, and the question is: does all browsers support this `PerformanceResourceTiming` interface?
If yes, we can force the players to rely indeed on this interface to retrieve the request metrics instead.

Reply:
Chrome since 29, but Safari came much later at version 11. I think we can use the `ResourceTiming` interface in the PR for now.